doc/api/repl.md

```diff
@@ -178,34 +174,6 @@ function myEval(cmd, context, filename, callback) {
 repl.start({prompt: '> ', eval: myEval});
 ```
 
-#### Recoverable Errors
```

Comment:
@TimothyGu we discussed this option [here](https://github.com/nodejs/node/issues/8195). <kbd>Shift</kbd>+<kbd>Enter</kbd> is not possible